iroh-net/src/discovery/pkarr.rs

```diff
@@ -174,6 +181,48 @@ impl PublisherService {
     }
 }
 
+/// Resolve node info using a pkarr relay.
```

Comment:
Would be good to expand the docs a little here, maybe with a link to pkarr or somewhere that explains what a pkarr relay is.